src/views/Project/index.jsx

```diff
@@ -288,7 +288,13 @@ export default class Project extends Component {
           bugzillaData.bugs.edges.map(edge => edge.node).map(bug => ({
             assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
             project: bug.component,
-            tags: bug.keywords || [],
+            tags: [
+              ...(bug.keywords || []),
+              ...(
+                (bug.whiteboard && bug.whiteboard.match(/\[.+?\]/g)) ||
```

Comment:
should we display other whiteboard tag / just language (the one that starts with lang=)?

Reply:
<img width="1680" alt="screen shot 2018-07-16 at 1 41 48 pm" src="https://user-images.githubusercontent.com/24544912/42746344-8d9164d0-8901-11e8-8baf-c258d20b3e20.png">
when we display everything from whiteboard, some of the tags will be a duplicated, e.g good-first-bug and good first bug, however it cant be detected since one uses space and the other use -